portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -79,18 +80,24 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType(contentType);
     String name = getParameter(request, "name", "Anonymous");
-    String comment = getParameter(request, "comment", "");
+    String comment = getParameter(request, "text", (String) null);
+    String userEmail = getParameter(request, "user", (String) null);
     Date createDate = new Date();
 
     if (comment == "" || comment == null) {
```

Comment:
I think that if the `text` parameter is missing, the `== ""` comparison will yield a NullPointerException.

Reply:
What Michal meant is:
```suggestion
    if (comment == null || comment == "" ) {
```

Reply:
Yes, I confirm this is what I meant.